### DISCUSSION TITLE
Can I instruct poetry to only use its cache?

### DISCUSSION THREAD

**Discussion by halloleo** [python-poetry/poetry#6767] [Q&A] [answered]
If I need to recreate a virtual environment while I'm offline, it would be great to instruct `poetry add` and `poetry install` to _only_ look in the cache `~/.cache/pypoetry` and not try to download stuff.

Is this possible?

**Comment by neersighted** [comment, marked as answer]
Not currently; this is tracked as a feature request in https://github.com/python-poetry/poetry/issues/2184.

**Reply by halloleo** [reply]
Thanks for the concise answer!
